config/settings.yml

```diff
@@ -59,6 +60,8 @@ DOI:
 HARVESTER:
   USE_MIDDLE_NAME: true
   USE_AUTHOR_IDENTITIES: false
+  UPDATE_TIMEFRAME: 4week # default WoS symbolicTimeSpan for updated authors/biweekly harvests
```

Comment:
These are attributes of WOS harvest only, not `HARVESTER`, right?

Reply:
yeah that's right